test/fixtures/account-token.js

```diff
@@ -0,0 +1,69 @@
+/*
+  This fixture creates a new application, directory,
+  and account.  It issues an Oauth request for the
+  account, resulting in an access token and refresh
+  token in the token collections for the account
+ */
+
+var helpers = require('../it/helpers');
+var OAuthPasswordGrantRequestAuthenticator = require('../../').OAuthPasswordGrantRequestAuthenticator;
+
+function AccountAccessTokenFixture(){
+  return this;
+}
+AccountAccessTokenFixture.prototype.before = function before(done) {
+  var self = this;
+  helpers.getClient(function(_client){
+
+    self.client = _client;
+    helpers.createApplication(function(err,app){
+      if (err) {
+        return done(err);
+      }else{
```

Comment:
Personal style, I'm okay with switching to no-else block.  I think @rdegges prefers that style?